contentcuration/contentcuration/frontend/channelEdit/components/edit/ActivityDuration.vue

```diff
@@ -167,6 +167,20 @@
       handleInput(value) {
         this.$emit('input', this.convertToSeconds(value));
       },
+      convertToHHMMSS(totalSeconds) {
+        if (totalSeconds !== null && Number.isInteger(totalSeconds)) {
+          const hours = Math.floor(totalSeconds / 3600)
```

Comment:
This method will give the same result and seems a little simpler: https://stackoverflow.com/a/25279399

Reply:
That implementation breaks when input seconds are greater than or equal to 86400 seconds.

Reply:
An excellent point!